djangocms_content_expiry/templates/djangocms_content_expiry/multiselect-filter.html

```diff
@@ -1,4 +1,9 @@
 {% load i18n %}
+<style>
+input[type="checkbox"]:checked + label {
```

Comment:
This should be added as a css: https://docs.djangoproject.com/en/3.2/ref/forms/widgets/#styling-widget-classes

Reply:
@Aiky30 struggled to merge the main changes into my existing branch so in order to not waste more time, I've recreated the PR

https://github.com/EligaServices/djangocms-content-expiry/pull/20